### ISSUE TITLE
Entering Occurrence Data - Taxon

### ISSUE THREAD

**Issue by brookerichards** [dbca-wa/wastd#308]
When entering the species name, the drop down options didn’t appear (side note - I left the computer to do something else and then came back later). This is possibly because the session timed out? After refreshing the page they auto appeared again as I started to write in the species name. 

Reproducible example:
1. 'Report Occurrence'
2. Leave computer for x amount of time
3. Go to continue entering data and taxon
4. 'No results found' in taxon field

**Comment by florianm** [comment]
Correct, the auto-lookup expires after a while. This should be mentioned in the docs:

How to enter occurrences: Enter the taxonomic/vernacular name within XX seconds to preview available options, else the widget times out and requires a page reload.

**Comment by florianm** [comment]
- [ ] Update help text
- [ ] Add to docs
